components/net/lib.rs

```diff
@@ -10,7 +10,7 @@
 #![feature(plugin)]
 #![plugin(plugins)]
 
-#![deny(unsafe_code)]
+//#![deny(unsafe_code)]
```

Comment:
Not sure, whether we use unsafe for StreamedResponse, which is required only for unit test.